beacon-chain/core/feed/operation/events.go

```diff
@@ -19,6 +19,9 @@ const (
 
 	// SyncCommitteeContributionReceived is sent after a sync committee contribution object has been received.
 	SyncCommitteeContributionReceived
+
+	// BLSToExecutionChangeReceived is sent after a BLS to execution change object has been received from the outside world
```

Comment:
this comment is not entirely accurate as we emit these when the local node operator submit changes as well